modules/hostap/Kconfig

```diff
@@ -164,7 +164,6 @@ endchoice
 
 config WIFI_NM_WPA_SUPPLICANT_CRYPTO_MBEDTLS_PSA
 	bool "Crypto Platform Secure Architecture support for WiFi"
-	default y if WIFI_NM_WPA_SUPPLICANT_CRYPTO_ALT
```

Comment:
@fengming-ye hope this is fine, not every implementation supports PSA, so, this should be explicitly enabled.

Reply:
yes, make sense